src/webots/nodes/WbTrack.cpp

```diff
@@ -468,6 +468,9 @@ void WbTrack::updateAnimatedGeometriesAfterFinalization(WbBaseNode *node) {
 void WbTrack::updateAnimatedGeometries() {
   clearAnimatedGeometries();
 
+  if (mWheelsList.size() <= 0)
```

Comment:
The size() of a QVector (alias QList) cannot be negative:
```suggestion
  if (mWheelsList.isEmpty())
```